jscomp/ml/translcore.ml

```diff
@@ -769,7 +786,10 @@ and transl_exp0 (e : Typedtree.expression) : Lambda.lambda =
           arity_opt
         else
           None in
-      transl_apply ~inlined ~uncurried_partial_application (transl_exp funct) oargs e.exp_loc
+      let is_tagged_template, _ =
+        Translattribute.get_and_remove_tagged_template_attribute e
```

Comment:
This (and the other question about `translcore.ml`) - are we satisfied with the current state or should we try and see if we can make it less invasive somehow? Would the solution @cristianoc proposes work instead (ignore attribute in the backend)?

Just making sure we've exhausted the various options. cc @tsnobip

Reply:
in which file(s) should we ignore the attribute? I have very little knowledge about the compiler internals, I've mostly reused the original PR and cleaned it, so if it can be improved, I'll do it with pleasure!